### DISCUSSION TITLE
How to use IRegister in asp.net 4.7.2 projects?

### DISCUSSION THREAD

**Discussion by netty2019** [MapsterMapper/Mapster#751] [Q&A]
My project is asp.net 4.7.2 MVC, Mapster version 7.2.0

Configuring IRegister does not work

    public class HjCollectDtlMapper : IRegister
    {
        public void Register(TypeAdapterConfig config)
        {
            config.ForType<SaveDtlInput, HjCollectDtl>()
                .Map(u => u.IsManual, p => p.IsManual ? "Y" : "N");
        }
    }

**Comment by vchc** [comment]
Do you scan for IRegister types at the startup?

`TypeAdapterConfig.GlobalSettings.Scan(typeof(HjCollectDtlMapper).Assembly);`

Or add them explicitly?

`TypeAdapterConfig.GlobalSettings.Apply(new HjCollectDtlMapper());`
